### PR TITLE
Add insert-block-test

### PR DESCRIPTION
**Pull request by andrew-t-james**

### CODE REVIEW

File: packages/blocks-ui/src/transforms.js [hunk 1]
```diff
@@ -132,6 +132,7 @@ export const reorderJSXBlocks = (code, drag) => {
 }
 
 export const insertJSXBlock = (code, { blocks, ...drag }) => {
+  console.log('blocks:', blocks)
```
Comment (johnletey) [inline packages/blocks-ui/src/transforms.js]:
```suggestion
```

@andrew-t-james This think was included by accident?

Comment (andrew-t-james) [reply packages/blocks-ui/src/transforms.js]: @johnletey yup my bad.

**Comment by andrew-t-james** [comment]
@johnletey FIxed

File: packages/blocks-ui/src/babel-plugins/insert-block.test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      mb: 0,
+      backgroundColor: 'rgba(206, 17, 38, 0.05)',
+      fontSize: '8pt'
+    }}>
+      {'"Unexpected template param undefined"'}
+    </pre>
+  </div></Blocks.Root>`)
+})
+
+test('should insert block when no destination defined', () => {
```
Comment (johno) [inline packages/blocks-ui/src/babel-plugins/insert-block.test.js]:
```suggestion
test('should not insert block when no destination defined', () => {
```

**Pull request merged**